qrenderdoc/Windows/EventBrowser.cpp

```diff
@@ -103,25 +103,25 @@ void EventBrowser::OnLogfileLoaded()
   framestart->setData(COL_CURRENT, Qt::UserRole, QVariant(false));
   framestart->setData(COL_FIND, Qt::UserRole, QVariant(false));
   framestart->setData(COL_BOOKMARK, Qt::UserRole, QVariant(false));
-  framestart->setData(COL_SELECT_EID, Qt::UserRole, QVariant(0));
+  framestart->setData(COL_LAST_EID, Qt::UserRole, QVariant(0));
 
   uint lastEID = AddDrawcalls(frame, m_Ctx->CurDrawcalls());
-  frame->setData(COL_EID, Qt::UserRole, QVariant(lastEID));
-  frame->setData(COL_SELECT_EID, Qt::UserRole, QVariant(lastEID));
+  frame->setData(COL_EID, Qt::UserRole, QVariant(0));
+  frame->setData(COL_LAST_EID, Qt::UserRole, QVariant(lastEID));
 
   ui->events->insertTopLevelItem(0, frame);
 
   ui->events->expandItem(frame);
 
-  m_Ctx->SetEventID(this, lastEID);
+  m_Ctx->SetEventID(this, 0, lastEID);
```

Comment:
The API inspector handles having a null draw call so that being 0 doesn't cause a problem, but yes, I think it probably makes more sense to use lastEID there (so the focus is entirely on the last event of the frame to begin with).